ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5451,6 +5496,17 @@ HWVM:
   implements:
   - OPERATIONAL
 
+HWVVM:
+  guid: "3e193668-77e3-454e-9153-fdc3475d9012"
+  description: "Heating water valve voltage monitoring."
+  is_abstract: true
+  opt_uses:
+  - heating_water_flowrate_sensor
+  - heating_water_valve_percentage_sensor
```

Comment:
can `HWVM` be used here instead?

Reply:
Looks like HWVM already exists: https://github.com/google/digitalbuildings/blob/1e4a4fb7b851553d0d7184ae23e3c9dd19637a10/ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml#L5462

Reply:
Yes, I'm wondering if you can just use that type rather than creating a new one and just mark the percentage sensor as missing in the building config.

Reply:
@shambergoldstein My mistake. there is a correction. These should be voltages, not percentage. I have corrected it